### ISSUE TITLE
Create a new parent element for cloud and scene cover as well as other satellite related elements

### ISSUE THREAD

**Issue by steingod** [steingod/mmd#41]
Consider to collect satellite related elements in a parent tag. Currently satellite specific tags are availabl for mmd:cloud_cover and mmd:scene_cover. Consider to collect these in a parent tag. A potential implementation is provided below for discussion.

'
<mmd:satellite_features>
  <mmd:orbit_absolute></mmd:orbit_absolute>
  <mmd:orbit_relative></mmd:orbit_relative>
  <mmd:cloud_cover>30</mmd:cloud_cover>
  <mmd:scene_cover>5</mmd:scene_cover>
</mmd:satellite_features>
'
This could potentially be part of platform as well. inclination, orbital plane, ascending/descending, etc should probably be part of platform since they are features of the platform?

**Comment by ferrighi** [comment]
@hevgyrt do you have comments on this?

**Comment by hevgyrt** [comment]
The parent element name could potentially be changed to be more generic. Like 
`<mmd:observational_data type="EO"> 
    ...
</mmd:observational_data> `

where the attribute "type" could be e.g. EO (earth obs.), IS (in-situ), or be described more granular. I would also argue to include ascending/descending into this element since it is an important characteristic of the observation.

**Comment by steingod** [comment]
I see the point, but we could complicate it too much. Perhaps better use specific tags for type without attributes? The content (sub elements) would be so different... I agree that more on platform characteristics could be added. Ascending/descending, sensor characteristic could be another. Any suggestion for specific elements and sub coding of these? 

That being said I think it is most important to get moving and rather extend later if necessary, but what is there should be useful/relevant.

**Comment by hevgyrt** [comment]
I agree. I would then suggest

```
<mmd:satellite_ancillary> \n 
\t<mmd:orbit_absolute>456</mmd:orbit_absolute> # (type integer)
\t<mmd:orbit_relative>22</mmd:orbit_relative> # (type integer)
\t<mmd:orbit_direction>ascending</mmd:orbit_relative> # (type string)
\t<mmd:cloud_cover>30</mmd:cloud_cover> # (type float)
\t<mmd:scene_cover>5</mmd:scene_cover> # (type float)
</mmd:satellite_ancillary>
```
I think you should decide on the final structure, @steingod .

**Comment by steingod** [comment]
BTW should this somehow be merged with the platform and instrument tags? Does that make sense or just complicating? platform and instrument are currently primarily used for the remote sensing setups.

**Comment by hevgyrt** [comment]
I agree that option 2 could potentially make things more easy. At the same time, the elements listed above is neither part of platform or instrument but ancillary data to the two.. We should however decide in order to get moving.

**Comment by ferrighi** [comment]
I commented in #13, and apparently it overlaps with this issue. I am not having a good overview of remote sensing data, but I think that integrating in platform/instrument would make more sense. Couldn't these entries be part of the platform, when the platform is a satellite?

**Comment by steingod** [comment]
I sure does, I responded to your comment as well. Concerning where these things belong, orbit is part of platform characteristics in my opinion, cloud and scene coverage not, those are more product related, but could be added as a flat structure. One option could be
```
<mmd:platform>
    <mmd:short_name>S1A</mmd:short_name>
    <mmd:long_name>Sentinel-1A</mmd:long_name>
    <mmd:resource>https://www.wmo-sat.info/oscar/satellites/view/396</mmd:resource>
     <mmd:instrument>
         <mmd:short_name>SAR-C SAR</mmd:short_name>
         <mmd:long_name>Synthetic Aperture Radar (C-band)</mmd:long_name>
         <mmd:resource>https://www.wmo-sat.info/oscar/instruments/view/450</mmd:resource>
         <mmd:mode>IW
            <mmd:polarisation>VV VH</mmd:polarisation>
         </mmd:mode>
    </mmd:instrument>
</mmd:platform>
```
Then we can remove instrument as it is today. It may have consequences for how we relate to WIGOS metadata for stations where we currently say we  use related_information, but we could refer to OSCAR information in a similar manner to what we have done for satellite here.

Any preferences?

**Comment by hevgyrt** [comment]
As we discussed briefly, I agree with the above comment and suggest the following structure for the platform element (including instrument and ancillary information):

`
<mmd:platform>
    <mmd:short_name>S1A</mmd:short_name>
    <mmd:long_name>Sentinel-1A</mmd:long_name>
    <mmd:resource>https://www.wmo-sat.info/oscar/satellites/view/396</mmd:resource>
    <mmd:orbit_relative>121</mmd:orbit_relative>
    <mmd:orbit_absolute>62391</mmd:orbit_absolute>
    <mmd:orbit_direction>ascending</mmd:orbit_direction>
    <mmd:instrument>
         <mmd:short_name>SAR-C SAR</mmd:short_name>
         <mmd:long_name>Synthetic Aperture Radar (C-band)</mmd:long_name>
         <mmd:resource>https://www.wmo-sat.info/oscar/instruments/view/450</mmd:resource>
         <mmd:mode>IW
            <mmd:polarisation>VV VH</mmd:polarisation>
         </mmd:mode>
    </mmd:instrument>
    <mmd:ancillary>
         <mmd:cloud_coverage>23.4</mmd:cloud_coverage>
         <mmd:scene_coverage>77</mmd:scene_coverage>
         <mmd:timeliness>NRT</mmd:timeliness>
    </mmd:ancillary>
</mmd:platform>`

This makes it easy to extend the element with new observation/ancillary information which is not covered by the current version. All "ancillary" information should be optional.

**Comment by steingod** [comment]
This sounds good to me, if there are no objections I would like to create a PR on this.

**Comment by steingod** [comment]
A PR is available in #62.
